test/e2e/framework/volume_util.go

```diff
@@ -398,10 +398,11 @@ func VolumeTestCleanup(f *Framework, config VolumeTestConfig) {
 		ExpectNoError(err, "Failed to delete client pod: %v", err)
 	}
 
+	if err := f.WaitForPodTerminated(config.Prefix+"-client", ""); !apierrs.IsNotFound(err) {
```

Comment:
This should actually be waiting for the pod to be deleted.  Terminated only means that the containers have been stopped, but not that the volumes have been unmounted.  Pod deletion indicates that the volumes have been unmounted.

Reply:
@msau42 

Fixed as suggested. If you meant to remove ```WaitForPodTerminated```, I will do so.

I kept both of them in order to keep maximum time to wait for the status the same. There will be no big difference in actual time to wait by doing so, for these functions will check the status immediately. And will improve the second wait time as it will quit waiting as soon as they find the pod no longer exists.

PTAL